site/docs/builder.Dockerfile

```diff
@@ -20,4 +20,9 @@ ENV PATH "/root/.local/bin:${PATH}"
 # Python version. If that information is not read, pip installs the latest
 # version, which then fails to run.
 RUN python3 -m pip install --user -U pip "setuptools<66.0.0"
-RUN pip3 install --user -r python-requirements.txt
+# Copy in-tree packages
+RUN mkdir -p util/py/install
+COPY util/py/install/fix_imports util/py/install/fix_imports
```

Comment:
Out of curiosity, why do we need `fix_imports` to run the `pip3 install` command?

Reply:
Good question. Our `Dockerfile`s only use `*-requirements.txt` files to install what they need. Since `fix_imports` is an in tree dependency we need to put it in the container during the build.

Reply:
BTW, I also considered publishing this as a separate repo but wanted to keep things simple. Unfortunately pip doesn't seem to support installing from a sub-folder of a git repo.